tools/gen_addon_readme.py

```diff
@@ -282,8 +289,13 @@ def gen_one_addon_index(readme_filename):
               help="git commit changes to README.rst, if any.")
 @click.option('--gen-html/--no-gen-html', default=True,
               help="Generate index html file.")
+@click.option('--lang', default='',
+              help="Target language file code (default is English)"
+                   "List of supported codes: "
+                   "[(--lang=es for Español),]")
```

Comment:
```suggestion
                   "[(--lang=es for Spanish),]")
```

Reply:
Somehow feels short: you should use the country+lang code.
What about MX,etc...